### PR TITLE
tidb-lightning: update docs

### PR DESCRIPTION
**Pull request by kennytm**
### What is changed, added or deleted? (Required)

Incorporate changes for these new features since last time:

* TLS config
* TiKV Importer now supports listening on 0.0.0.0
* TiKV Importer no longer requires Pushgateway for Prometheus
* CSV strict-format config for optional parallel speed up
* TiDB's default SQL mode is no longer strict
* Several new CLI parameters are exposed

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

    This PR involves changes introduced from:

    * pingcap/tidb-lightning#255
    * pingcap/tidb-lightning#270
    * pingcap/tidb-lightning#272
    * pingcap/tidb-lightning#287
    * pingcap/tidb-lightning#316
    * tikv/importer#40
    * tikv/importer#46

### CODE REVIEW

File: tidb-lightning/monitor-tidb-lightning.md [hunk 1]
```diff
@@ -44,14 +24,21 @@ pprof-port = 8289
...
+
+You need to configure Prometheus to make it discover the servers. For instance, you can directly add the server address to the `scrape_configs` section:
 
 ```yaml
 ...
 scrape_configs:
   - job_name: 'tidb-lightning'
     static_configs:
-      - targets: ['192.168.20.10:8289']
+      - targets: ['192.168.20.10:8289', '192.168.20.9:8286']
```
Comment (kissmydb) [inline tidb-lightning/monitor-tidb-lightning.md]: please modify scrape to:
`  - job_name: "tikv-importer"
    static_configs:
    - targets: ['172.16.5.169:8286']
  - job_name: 'lightning'
    static_configs:
    - targets: ['172.16.5.169:8289']`

**Comment by yikeke** [comment]
PTAL @3pointer

**Pull request merged**